src/signals/incident/components/IncidentPreview/index.test.js

```diff
@@ -1,10 +1,71 @@
-// import React from 'react';
-// import { shallow } from 'enzyme';
+import React from 'react';
+import { shallow, mount } from 'enzyme';
 
-// import IncidentPreview from 'index';
+import { Wizard, WithWizard } from 'react-albus';
+
+import PreviewComponents from '../../components/IncidentPreview/components/';
+import IncidentPreview from './index';
 
 describe('<IncidentPreview />', () => {
-  it('Expect to have unit tests specified', () => {
-    expect(true).toEqual(true);
+  let props;
+
+  beforeEach(() => {
+    props = {
+      incidentContainer: {
+        incident: {
+          phone: '0666 666 666',
+          email: 'devil@hell.com'
```

Comment:
Not the best test email for a public repository

Reply:
gedaan